README.md

```diff
@@ -1,63 +1,29 @@
-# epitools
+# EpiTools
 
-[![License](https://img.shields.io/pypi/l/epitools.svg?color=green)](https://raw.githubusercontent.com/epitools/epitools/main/LICENSE)
+[![Licence](https://img.shields.io/pypi/l/epitools.svg?color=green)](https://raw.githubusercontent.com/epitools/epitools/main/LICENCE.md)
 [![PyPI](https://img.shields.io/pypi/v/epitools.svg?color=green)](https://pypi.org/project/epitools)
 [![Python Version](https://img.shields.io/pypi/pyversions/epitools.svg?color=green)](https://python.org)
 [![tests](https://github.com/epitools/epitools/actions/workflows/test.yml/badge.svg)](https://github.com/epitools/epitools/actions/workflows/test.yml)
 [![coverage](https://coveralls.io/repos/github/epitools/epitools/badge.svg?branch=main)](https://coveralls.io/github/epitools/epitools?branch=main)
 [![napari hub](https://img.shields.io/endpoint?url=https://api.napari-hub.org/shields/epitools)](https://napari-hub.org/plugins/epitools)
 
-EpiTools Plugin
-
----
-
-This [napari] plugin was generated with [Cookiecutter] using [@napari]'s [cookiecutter-napari-plugin] template.
-
-<!--
-Don't miss the full getting started guide to set up your new package:
-https://github.com/napari/cookiecutter-napari-plugin#getting-started
-
-and review the napari docs for plugin developers:
-https://napari.org/plugins/index.html
--->
-
 ## Installation
 
-`epitools` is not yet available to install from PyPI or napari-hub.
-
-To install, please clone the repository and do a local install using `pip`:
+The recommended way to install `EpiTools` is via
+[pip](https://pypi.org/project/pip)
 
+```sh
+pip install epitools
 ```
-git clone https://github.com/epitools/epitools.git
-cd epitools
-python -m pip install .
+
+To install the latest development version of `EpiTools` clone this repository
+and run
+
+```sh
+pip install -e .
 ```
 
 ## Contributing
 
 Contributions are very welcome. Tests can be run with [tox], please ensure
 the coverage at least stays the same before you submit a pull request.
-
-## License
-
-Distributed under the terms of the [BSD-3] license,
-"epitools" is free and open source software
-
-## Issues
-
-If you encounter any problems, please [file an issue] along with a detailed description.
```

Comment:
we could probably keep the `Issues` sub-heading with a link to the github issue tracker (again so this would appear in the docs)

Reply:
Will restore